### ISSUE TITLE
Moving to the "facility access" subtab for users is not possible using only the keyboard

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4099]
## Area or Component
Manage users - "facility access" sub-nav menu item

## Quick Summary
It's not possible to navigate to the "facility access" sub-page within a user profile by the keyboard alone - you must to click on it.

### How to Replicate
1. Go to "manage users" page
2. Tab through the page and try to reach facility access 

## Accessibility Violation
Action cannot be performed by keyboard alone: https://cdc-devtools.dequecloud.com/rule-help/en/keyboard-inaccessible

## Action Requested
The sub-nav is coded as a list item, but the items are just `<span>` items instead of a button or link. They should be changed to an interactable element instead.

## Screenshots (optional)
<img width="415" alt="Screen Shot 2022-07-27 at 4 40 59 PM" src="https://user-images.githubusercontent.com/80282552/181390635-1f4ff2c0-6539-4495-9038-b9f8d7bfcc80.png">

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=1647999156&range=2:2

**Comment by emmastephenson** [comment]
A related issue from the audit:
`The element's role is missing or is not appropriate for the element's function.`
The "user information" and "facility access" tabs are coded as list items, but should really be a tab or button. We should be able to fix all these issues at once.
